python/main.py

```diff
@@ -332,9 +335,361 @@ def recognized(evt):
         print(f"       {idx + 1:02d}:\t{word.word:16s}{word.accuracy_score:.1f}\t\t{phoneme_accuracy_score}")
 
 
+def read_wave_header(file_path):
+    with wave.open(file_path, 'rb') as audio_file:
+        framerate = audio_file.getframerate()
+        bits_per_sample = audio_file.getsampwidth() * 8
+        num_channels = audio_file.getnchannels()
+        return framerate, bits_per_sample, num_channels
+
+
+def push_stream_writer(stream, filenames):
+    # The number of bytes to push per buffer
+    n_bytes = 3200
+    try:
+        for filename in filenames:
+            wav_fh = wave.open(filename)
+            # Start pushing data until all data has been read from the file
+            try:
+                while True:
+                    frames = wav_fh.readframes(n_bytes // 2)
+                    # print('read {} bytes'.format(len(frames)))
+                    if not frames:
+                        break
+                    stream.write(frames)
+                    time.sleep(.1)
+            finally:
+                wav_fh.close()
+    finally:
+        stream.close()
+
+
+def chatting_from_file():
+    """Performs pronunciation assessment asynchronously from an audio file and enable prosody/content assessment
+        with continuous mode.
+        See more information at https://aka.ms/csspeech/pa"""
+
+    topic = "YourOwnTopic"
```

Comment:
You have specified the input file. So please change `YourOwnTopic` to the real topic.

Reply:
have changed to sample topic